documentation/src/test/java/org/hibernate/search/documentation/mapper/orm/indexing/HibernateOrmBatchJsr352IT.java

```diff
@@ -80,14 +80,14 @@ public void simple() throws Exception {
 	@Test
 	public void hql() throws Exception {
 		// tag::hql[]
-		Properties jobProps = MassIndexingJob
+		Properties jobProps = MassIndexingJob // <1>
 				.parameters()
```

Comment:
```suggestion
		Properties jobProps = MassIndexingJob.parameters() // <1>
```

Reply:
Did you miss this suggestion or decide against applying it?

Reply:
Actually I thought I applied for the change... Doing now...